proto/beacon/rpc/v1/services.proto

```diff
@@ -68,6 +68,7 @@ message AttestationDataResponse {
     uint64 justified_epoch = 3;
     bytes justified_block_root_hash32 = 4;
     ethereum.beacon.p2p.v1.Crosslink latest_crosslink = 5;
+    uint64 slot = 6;
```

Comment:
Why is this needed?

Reply:
Attester needs to know the head slot and it has no one of knowing unless we include head slot in one of the Response

Reply:
can we call this head slot then